src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/service/GeneralPartnerServiceValidateTest.java

```diff
@@ -184,6 +236,7 @@ private GeneralPartnerDao createPersonDao() {
         dataDao.setForename("Jack");
         dataDao.setSurname("Jones");
         dataDao.setDateOfBirth(LocalDate.of(2000, 10, 3));
+        dataDao.setDateEffectiveFrom(LocalDate.of(2023, 10, 3));
```

Comment:
had to add the effective date as test would always include error for effective date